### DISCUSSION TITLE
`altsrc` housekeeping

### DISCUSSION THREAD

**Discussion by abitrolly** [urfave/cli#1934] [General]
I've changed description for https://github.com/urfave/cli-altsrc, added PR to fix links and provide an example.

The example and docs, however, don't explain if `altsrc` reads each config file each time for each variable, or config files are read only once.

Does it support live updates? Or app needs to be restarted for new config values to be updated.

**Comment by dearchap** [comment]
No live updates.

**Reply by abitrolly** [reply]
What about caching reads?

And also names independent from flag names https://github.com/urfave/cli-altsrc/issues/2
